scripts/install.sh

```diff
@@ -32,6 +32,19 @@ ln -sf /etc/opensafely/profile /etc/profile.d/opensafely.sh
 ln -sf /etc/opensafely/sudoers /etc/sudoers.d/opensafely
 ln -sf /etc/opensafely/ssh.conf /etc/ssh/sshd_config.d/99-opensafely.conf
 
+chmod 0755 /etc/opensafely/{banner,bashrc}
+
+# add login banner
+
+ln -sf /etc/opensafely/banner /etc/update-motd.d/00-opensafely-banner
+# add our bashrc config
+if ! grep -q "etc/opensafely/bashrc" /etc/bash.bashrc; then
+    echo "test -f /etc/opensafely/bashrc && . /etc/opensafely/bashrc" >> /etc/bash.bashrc
+fi
+
+# reduce motd noise
+chmod a-x /etc/update-motd.d/{10-help-text,50-motd-news,90-updates-available,91-contract-ua-esm-status,91-release-upgrade}
```

Comment:
nice idea - should we keep `90-updates-available` though?

Reply:
(actual question - should we be doing updates more often)

Reply:
I don't want folks to be prompted when the log in to do ad hoc this though - it should be deliberate (or automated).